boards/arm/ubx_evkannab4_nrf52833/doc/index.rst

```diff
@@ -0,0 +1,220 @@
+
+The numbering of the pins on the module and EVK do not follow the GPIO
+numbering on the nRF52833 SoC. Please see the `ANNA-B40 Data Sheet`_ for
+information on how to map ANNA-B40 pins to the pin numbering on the
+nRF52833 SoC.
+
+The reason for this is the u-blox module family concept where different
+modules share the same pinout and can be interchanged, see
+`ANNA module family Nested design`_.
+
+Programming and Debugging
+*************************
+
+Applications for the ``ubx_evkannab4_nrf52833`` board configuration can be
+built and flashed in the usual way (see :ref:`build_an_application`
+and :ref:`application_run` for more details); however, the standard
+debugging targets are not currently available.
+
+Flashing
+========
+
+Build and flash applications as usual (see
+:ref:`build_an_application` and :ref:`application_run` for more details)
+
+Here is an example for the :ref:`hello_world` application.
+
+Open a terminal program to the USB Serial Port installed when connecting
+the board and listen for output.
+
+Settings: 115200, 8N1, no flow control.
+
+Then build and flash the application in the usual way.
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/hello_world
+   :board: ubx_evkanna4_nrf52833
+   :goals: build flash
+
+Debugging
+=========
+
+Refer to the :ref:`nordic_segger` page to learn about debugging  boards
+containing a Nordic Semiconductor chip with a Segger IC.
+
+
+Testing the LEDs and buttons in the EVK ANNA-B402
+*************************************************
+
+There are 2 samples that allow you to test that the buttons (switches)
+and LEDs on the board are working properly with Zephyr:
+
+.. code-block:: console
+
+   samples/basic/blinky
+   samples/basic/button
+
+You can build and flash the examples to make sure Zephyr is running
+correctly on your board. The button and LED definitions can be found in
+:zephyr_file:`boards/arm/ubx_evkannab4_nrf52833/ubx_evkannab4_nrf52833.dts`.
+
+Note that the buttons on the EVK-ANNA-B4 are marked SW1 and SW2, which
+are named sw0 and sw1 in the dts file.
+Also note that the SW1 button and the green LED are connected on HW level.
+
+Using UART1
+***********
+
+The following approach can be used when an application needs to use
+more than one UART for connecting peripheral devices:
+
+1. Add device tree overlay file to the main directory of your application:
+
+   .. code-block:: console
+
+      $ cat ubx_evkannab4_nrf52833.overlay
+      &uart1 {
+        compatible = "nordic,nrf-uarte";
+        current-speed = <115200>;
+        status = "okay";
+        tx-pin = <14>;
+        rx-pin = <16>;
+      };
+
+   In the overlay file above, pin P0.16 is used for RX and P0.14 is used for TX
+
+2. Use the UART1 as ``device_get_binding(DT_LABEL(DT_NODELABEL(uart1)))``
+
+
+See :ref:`set-devicetree-overlays` for further details.
+
+Selecting the pins
+==================
+To select the pin numbers for tx-pin and rx-pin:
+
+.. code-block:: console
+
+   tx-pin = <pin_no>
```

Comment:
```suggestion
==================

To select the pin numbers for tx-pin and rx-pin::

   tx-pin = <pin_no>
```

Reply:
This part is similar in many of the boards (nrf52833dk_nrf52833, nrf52840dk_nrf52840 etc), should I change here or should there be a common change for all of them?

Reply:
Done